aqua/cool-retro-term/Portfile

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           qmake5 1.0
+PortGroup           github 1.0
+
+github.setup        Swordfish90 cool-retro-term 1.1.1
+revision            0
+
+categories          aqua shells
+platforms           darwin
+license             GPL-3+
+maintainers         {amake @amake} openmaintainer
+
+description         A good looking terminal emulator which mimics the old cathode display
+
+long_description    A terminal emulator which mimics the look and feel of the \
+                    old cathode tube screens. It has been designed to be \
+                    eye-candy, customizable, and reasonably lightweight.
+
+set dep_name        qmltermwidget
+set dep_version     48274c75660e28d44af7c195e79accdf1bd44963
+set dep_distname    ${dep_name}-${dep_version}
+
+master_sites        ${github.master_sites}:main \
+                    https://github.com/${github.author}/${dep_name}/tarball/${dep_version}:dep
+
+distfiles           ${distname}${extract.suffix}:main \
+                    ${dep_distname}${extract.suffix}:dep
+
+checksums           ${distname}${extract.suffix} \
+                        rmd160  3f0e4787b69802bcea7715c7cd92859e4a455c87 \
+                        sha256  67eb9c4db2bd4036fa54f7ab792194ca5d24ca7667a2ce534a2fc9ebaaf81bfa \
+                        size    1975797 \
+                    ${dep_distname}${extract.suffix} \
+                        rmd160  f1bb413b6d3300390edbd971344f5b84ded691ab \
+                        sha256  439627c6ee8aadfb6ca60c5559cef71f7757960a9913b91c9347d898cf84bc0b \
+                        size    219567
+
+qt5.depends_component  qtquickcontrols
+qt5.depends_runtime_component  sqlite-plugin
```

Comment:
Minor nitpick. I don't think the spacing here is honoring the mode line. Either that or the checksums above are not.

Reply:
I enjoy a nitpick :)

I use Emacs and the modeline doesn't do much to help indent to what I think you're talking about, which is aligning to column 20. All I can get it to do for me is indent 4 spaces per `c-basic-offset`.

I am always ambivalent about what to do with one-off lines like this that have keys so long as to prevent aligning to column 20. I feel like it's not really an improvement to break the line just for one value, so here I kept one line (with two spaces after the key because one is too small a break).

The checksums are following the [MacPorts Guide](https://guide.macports.org/index.html#reference.phases.checksum) example for multiple files. I think it makes sense because the value semantics are hierarchical.

Reply:
It's not about being at 20 spaces per se. It is every 4 spaces. So if it cannot align at 20 either move to the next line or bump out to 24 or 28 spaces.

Reply:
Got it. Thanks.